packages/amplify-cli/src/__tests__/commands/upgrade.pkg.test.ts

```diff
@@ -147,24 +159,24 @@ describe('run upgrade using packaged CLI', () => {
     await run(context_stub_typed);
 
     // validate
-    expect(fs_mock.move.mock.calls[0]).toMatchInlineSnapshot(`
+      expect(fs_mock.move.mock.calls[0]).toMatchInlineSnapshot(`
```

Comment:
The indentation looks off here and on a number of the lines below.

Reply:
My bad, sure will resolve this